src/actions/get-transfer-bounty-token-events.ts

```diff
@@ -0,0 +1,112 @@
+import db from "src/db";
+import logger from "src/utils/logger-handler";
+import {
+  BountyToken,
+  NetworkRegistry,
+  Network_v2,
+  Web3Connection,
+} from "@taikai/dappkit";
+import {
+  EventsProcessed,
+  EventsQuery,
+} from "src/interfaces/block-chain-service";
+import { leaderboardAttributes } from "src/db/models/leaderboard";
+import loggerHandler from "src/utils/logger-handler";
+
+export const name = "getTransferEvents";
+export const schedule = "*/60 * * * *";
+export const description = "retrieving bounty token transfer events";
+export const author = "MarcusviniciusLsantos";
+
+const { PUBLIC_WEB3_CONNECTION: web3Host, WALLET_PRIVATE_KEY: privateKey } =
+  process.env;
+
+export async function action(query?: EventsQuery): Promise<EventsProcessed> {
+  const eventsProcessed: EventsProcessed = {};
+
+  const settings = await db.settings.findAll({
+    where: { visibility: "public" },
+    raw: true,
+  });
+
+  if(!settings) logger.warn(`${name} Failed missing settings`);
+
+  const networkRegistry = settings.find(({ key }) => key === "networkRegistry");
+
+  if (!networkRegistry) {
+    logger.warn(`${name} Failed missing network registry`);
+  } else {
+
+    try {
+      const web3Connection = new Web3Connection({ web3Host, privateKey });
+      await web3Connection.start();
+
+      const registry = new NetworkRegistry(web3Connection, networkRegistry.value);
+      await registry.loadContract();
+  
+      let lastReadBlock = await db.chain_events.findOne({
+        where: { name: name },
+      });
+
+      const network = await db.networks.findOne({
```

Comment:
We don't need the network to be loaded, as we only need the transferEvents from the bounty-token itself, we can then correlate that bounty-id with the issue id which will give us the network name without having to find it first